src/analysis/normed_space/affine_isometry.lean

```diff
@@ -0,0 +1,441 @@
+/-
+Copyright (c) 2021 Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth
+-/
+import analysis.normed_space.add_torsor
+import analysis.normed_space.linear_isometry
+import linear_algebra.affine_space.affine_subspace
+
+/-!
+# Affine isometries
+
+In this file we define `affine_isometry 𝕜 P P₂` (notation: `P →ᵃⁱ[𝕜] P₂`) to be a affine isometric
```

Comment:
```suggestion
In this file we define `affine_isometry 𝕜 P P₂` (notation: `P →ᵃⁱ[𝕜] P₂`) to be an affine isometric
```
? (I'm not a native speaker, so dismiss if needed). Same thing several times below.

Reply:
Indeed, just search-and-replace from `linear_isometry`.  I'll fix.